src/main/resources/templates/headers.vm

```diff
@@ -18,5 +18,6 @@
 #parse("/templates/macros/report/hooks.vm")
 #parse("/templates/macros/report/duration.vm")
 #parse("/templates/macros/report/message.vm")
-#parse("/templates/macros/report/output.vm")
 #parse("/templates/macros/report/docstring.vm")
+#parse("/templates/macros/report/combinedOutput.vm")
+#parse("/templates/macros/report/differentOutputs.vm")
```

Comment:
I'd prefer to have one file per element so can you combine into out output..vm file as it used to be?